crates/bitwarden/src/crypto/enc_string.rs

```diff
@@ -165,6 +165,12 @@ impl FromStr for EncString {
 }
 
 impl EncString {
+    /// Synthetic sugar for mapping `Option<String>` to `Result<Option<EncString>>`
+    #[cfg(feature = "internal")]
+    pub(crate) fn try_from(s: Option<String>) -> Result<Option<EncString>, Error> {
```

Comment:
Should we name this something like try_from_optional so it can't be confused with the actual TryFrom trait?